test/parallel/test-cli-syntax.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const assert = require('assert');
+const spawnSync = require('child_process').spawnSync;
+const path = require('path');
+
+const common = require('../common');
+
+var node = process.argv[0];
+
+// test both sets of arguments that check syntax
+var syntax_args = [
```

Comment:
Can you spell this as `syntaxArgs`?  snake_case is not entirely prohibited in the test suite but the convention is camelCase.